docs/csharp/language-reference/operators/snippets/shared/NullCoalescingOperator.cs

```diff
@@ -57,12 +57,20 @@ private static void NullCoalescingAssignment()
         List<int> numbers = null;
         int? a = null;
 
+        Console.WriteLine((numbers is null)); // expected: true
+        // if numbers is null add 5 to numbers
```

Comment:
Let's change this comment to be more explicit about the action:

```suggestion
        // if numbers is null, initialize it. Then, add 5 to numbers
```